lib/bundler/spec_set.rb

```diff
@@ -99,6 +99,17 @@ def materialize(deps, missing_specs = nil)
       SpecSet.new(materialized.compact)
     end
 
+    def materialized_for_all_platforms
```

Comment:
it's just all the specs in the spec set, regardless of what platform they're for, in contrast to how `for` does platform filtering (and specifically different from how `materialize` calls `for` only for the current platform)